ui/app/src/Views/SilenceForm/Views.elm

```diff
@@ -81,45 +90,41 @@ createSilence maybeId silenceResult =
                 Nothing ->
                     "Create"
     in
-        case silenceResult of
-            Ok silence ->
-                a
+        case ( isFormValid form, silenceResult ) of
```

Comment:
If form is invalid, then the `silenceResult` should be `Err`, we don't need two indicators.